test/system/comment_test.rb

```diff
@@ -74,4 +74,40 @@ def setup
     assert_equal( comment_preview_button.text, "Preview" )
   end
 
+  test 'comment image upload' do
```

Comment:
There should be a test for exceeding image size too

Reply:
The question is, do you have a limit set? I've uploaded 60mb image without a problem.

Reply:
If you don't display some kind of error message or something, then I can't test it.

Reply:
Kindly check. We need to set the limit if not set yet. pinging @jywarren for limit